core/src/test/java/org/opentosca/toscana/core/api/upload/UploadTest.java

```diff
@@ -37,14 +37,14 @@ public void setUp() throws Exception {
         api = retrofit.create(TOSCAnaUploadInterface.class);
 
         springThread = new Thread(() -> Main.main(new String[]{
-            "--datadir=" + tmpdir.getAbsolutePath(),
+//            "--datadir=" + tmpdir.getAbsolutePath(),
```

Comment:
Is this [WIP]?
We have to use the BaseTest classes to ensure data cleanup. this `springThread` will produce some test files which are not cleaned up.

Reply:
The test is disabled!